### PR TITLE
Add WFM example in docs

### PR DESCRIPTION
**Pull request by nvaytet**

### CODE REVIEW

File: docs/wfm-stitching.ipynb [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+    "    ]\n",
+    "    new_choppers = wfm_choppers + choppers[2:]\n",
+    "    model = tof.Model(source=source, choppers=new_choppers, detectors=detectors)\n",
+    "    res = model.run()\n",
+    "    results.append(res)\n",
+    "\n",
+    "invalid_frames = True\n",
+    "fact = 0.01\n",
+    "while invalid_frames:\n",
+    "    print(f\"Searching for bounds: threshold={fact}\")\n",
```
Comment (YooSunYoung) [inline docs/wfm-stitching.ipynb:229]: This part renders a bit funny.
Should it be rounded up?

image

File: docs/wfm-stitching.ipynb [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fcb52d39-56d6-4cfb-8bcd-be9f45f4cc70",
+   "metadata": {},
+   "source": [
+    "## Find WFM frame edges\n",
+    "\n",
+    "To compute the frame edges, we use one of the openings in the WFM choppers are a time,\n",
```
Comment (YooSunYoung) [inline docs/wfm-stitching.ipynb:192]: I didn't get this sentence. Especially `are a time` part.

**Review by YooSunYoung — APPROVED** [review]
I just have a couple of minor comments but it seems nice : D

And one more question regarding the target audience.
If the target audience already know about WFM and just wants to know more about the WFM stitching methodology, I think it is fine as it is, 
but if it is about introducing WFM stitching methodology, I think it'll be nice to add more diagrams or pictures to explain the concept instead of just showing the code. But that can be done in another time.

File: docs/wfm-stitching.ipynb (hunk 2 above)
Comment (nvaytet) [reply docs/wfm-stitching.ipynb:192]: Because it should be "we use one of the openings in the WFM choppers at a time" ;-)

**Comment by nvaytet** [comment]
> I just have a couple of minor comments but it seems nice : D
> 
> And one more question regarding the target audience. If the target audience already know about WFM and just wants to know more about the WFM stitching methodology, I think it is fine as it is, but if it is about introducing WFM stitching methodology, I think it'll be nice to add more diagrams or pictures to explain the concept instead of just showing the code. But that can be done in another time.

For the target audience, I wanted to link to the [old notebook](https://scipp.github.io/ess/techniques/wfm/introduction-to-wfm.html) we have that introduces the technique. However, there are 2 things to consider.

1. I would like to move that notebook from that repo to a new one (probably scippneutron), so I felt like adding a link only to change it later was not great.
2. That notebook uses the old `ess` code to perform the WFM correction to `tof`, and I didn't want to confuse users who see one way of doing it there, and another one here...

**Review by YooSunYoung — APPROVED** [review]

**Comment by YooSunYoung** [comment]
Then we can add more link/general explanation later...!

**Pull request merged**